client/rtc.go

```diff
@@ -191,15 +191,17 @@ func (c *Client) initRTCSession() error {
 
 	i := interceptor.Registry{}
 
-	statsInterceptorFactory, err := stats.NewInterceptor()
-	if err != nil {
-		return fmt.Errorf("failed to create stats interceptor: %w", err)
-	}
 	var statsGetter stats.Getter
-	statsInterceptorFactory.OnNewPeerConnection(func(_ string, g stats.Getter) {
-		statsGetter = g
-	})
-	i.Add(statsInterceptorFactory)
+	if c.cfg.EnableRTCMonitor {
```

Comment:
👍

Reply:
Yes, as part of my investigation I wanted to rule out this wasn't a problem so it made sense to only register this interceptor when the monitor is enabled. Either way, it ended up not being a perf issue.